integration/spark-common-test/src/test/scala/org/apache/carbondata/spark/testsuite/datamap/DataMapWriterSuite.scala

```diff
@@ -156,8 +156,7 @@ class DataMapWriterSuite extends QueryTest with BeforeAndAfterAll {
     CarbonProperties.getInstance()
       .addProperty("carbon.blockletgroup.size.in.mb", "16")
     CarbonProperties.getInstance()
-      .addProperty("carbon.number.of.cores.while.loading",
-        CarbonCommonConstants.NUM_CORES_DEFAULT_VAL)
+      .addProperty("carbon.number.of.cores.while.loading", "2")
```

Comment:
CarbonCommonConstants.NUM_CORES_LOADING

Reply:
CarbonCommonConstants.NUM_CORES_DEFAULT_VAL is removed, so setting 
CarbonCommonConstants.NUM_CORES_LOADING
 as 2. It is a test code.

Reply:
And @qiuchenjian : If you have a doubt in PR changes, you can ask it in comment sections. Don't add as a review comment. Just add as a comment.

Reply:
OK,  I mean you  can use CarbonCommonConstants.NUM_CORES_LOADING instead of "carbon.number.of.cores.while.loading"